docs/getting_started.rst

```diff
@@ -112,14 +116,40 @@ For whichever board you are using, make sure the lines in the appropriate sectio
 - **ssh_port**: The port used to open ``SSH`` communications between the host and FPGA board.
 - **ssh_user**: ``SSH`` username to use to login to the board.
 - **ssh_pwd**: Password for ``ssh_user`` to use to login to the board. Note that the ``fpga_config`` file supports the use of ``SSH`` keys (see :ref:`ssh-key`) as an alternate form of authentication.
-- **remote_script**: The location of the communication script on the FPGA board.
+- **remote_script**: The location of the main communication script on theFPGA board.
+- **dna_script**: The location of the script that extracts the unique device identifier (called device DNA)
 - **remote_tmp**: Temporary location used to store data as it is transferred between the host and FPGA board.
 - **udp_port**: The port used for UDP communications between the host and FPGA board.
 
 
 .. note::
    It should be noted that the FPGA board should be configured such that non-root users do not require a password to perform ``sudo`` commands. Refer to the respective FGPA board documentation for instructions on how to do this.
 
+
+Copy Protection
+===============
+
+Our hardware design (known as the bitstream) is locked to a specific device.
+Each bitstream is compiled with your unique board ID (called device DNA)
+and therefore you will need to provide this unique ID to us before we
+can compile and deliver your tailored bitstream.
+
+Reading Device DNA
+------------------
+To easily read your device DNA, first ensure you have setup your board
+according to its affiliated documentation and reviewed the
+``fpga_config`` file in this repository as per the `Configuration`_ section.
+
+Now simply run the ``dna_extractor.py`` script located in the ``nengo-fpga``
```

Comment:
This description will have to change slightly. It doesn't return a datestamped file any more. 
Also, I think the user should run the script from the `nengo-fpga` root directory, so that the dna files dont clutter the `nengo_fpga` directory.

i.e., (from the `nengo-fpga` root directory) `python nengo_fpga/dna_extractor.py pynq`

This will save the dna files to the `nengo-fpga` root directory as well.